packages/quantic/force-app/main/default/lwc/quanticDateFacet/quanticDateFacet.js

```diff
@@ -248,4 +276,52 @@ export default class QuanticDateFacet extends LightningElement {
   preventDefault(evt) {
     evt.preventDefault();
   }
+
+  /**
+   * Sets the focus on the target element.
+   */
+  setFocusOnTarget() {
+    this.focusShouldBeInFacet = false;
+    if (!this.focusTarget) {
+      return;
+    }
+    // eslint-disable-next-line default-case
+    switch (this.focusTarget.type) {
+      case 'facetHeader':
+        this.setFocusOnHeader();
+        return;
+      case 'facetValue':
+        if (this.focusTarget.value) {
+          const facetValueIndex = this.values.findIndex(
+            (value) => this.formattingFunction(value) === this.focusTarget.value
+          );
+          this.focusTarget.index = facetValueIndex >= 0 ? facetValueIndex : 0;
+          this.setFocusOnFacetValue();
+        }
+    }
+  }
+
+  /**
+   * Sets the focus on the target facet value.
+   */
+  setFocusOnFacetValue() {
+    const focusTarget = this.template.querySelectorAll('c-quantic-facet-value')[
+      this.focusTarget.index
+    ];
+    if (focusTarget) {
+      // @ts-ignore
+      focusTarget.setFocus();
+    }
+  }
+
+  /**
+   * Sets the focus on the facet header.
+   */
+  setFocusOnHeader() {
+    const focusTarget = this.template.querySelector('c-quantic-card-container');
```

Comment:
could also be `this.template.querySelector('c-quantic-card-container')?.setFocus();`